pkg/common/config.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+// ServerConfiguration is the config for connecting to a CRI server
+type ServerConfiguration struct {
+	// RuntimeEndpoint is CRI server runtime endpoint
+	RuntimeEndpoint string
+	// ImageEndpoint is CRI server image endpoint
+	ImageEndpoint string
+	// Timeout  of connecting to server
+	Timeout time.Duration
+	// Debug enable debug output
+	Debug bool
+}
+
+// GetServerConfigFromFile returns the CRI server configuration from file
+func GetServerConfigFromFile(configFileName, currentDir string) (*ServerConfiguration, error) {
+	serverConfig := ServerConfiguration{}
+	if _, err := os.Stat(configFileName); err != nil {
+		if !os.IsNotExist(err) {
+			return nil, fmt.Errorf("Failed to load config file: %v", err)
```

Comment:
Let's use the pkg/errors package here and in other locations as well:
```suggestion
			return nil, errors.Wrap(err, "load config file")
```

Reply:
Sure. I updated it for the files changed in this PR. I will open a new PR to cover the rest of the files in the project if thats ok? I want to avoid changing files not directly related to the fix.